common-test/src/main/java/datawave/common/test/utils/query/RangeFactoryForTests.java

```diff
@@ -4,7 +4,11 @@
 import org.apache.accumulo.core.data.PartialKey;
 import org.apache.accumulo.core.data.Range;
 
-public class RangeFactory {
+/**
+ * Utility class that generates ranges for unit tests.
+ */
+public class RangeFactoryForTests {
```

Comment:
Nitpick: I kind of hate this name. Not sure I have a better one. Maybe just TestRangeFactory or somthing. Not sure why but the ForTests is really bothering me